bluesky/tests/conftest.py

```diff
@@ -36,5 +37,6 @@ def motor_det(request):
 def db(request):
     """Return a data broker
     """
-    from databroker import temp_config, Broker
-    return Broker.from_config(temp_config())
+    from databroker.tests.utils import build_pymongo_backed_broker
+    db = build_pymongo_backed_broker(request)
```

Comment:
Another option is to switch this to the sqlite backend factory which already has the clean up attached to it.